site/app/templates/CreateCourseForm.twig

```diff
@@ -0,0 +1,99 @@
+<div class="content">
+    <h2>New Course -- Work In Progress</h2>
+
+    <form id="course-creation-form" action="{{ course_creation_url }}"
+          method="post" enctype="multipart/form-data">
+        <div class="option row">
+            <div class="option-desc col-md-6">
+                <label for="course_semester">
+                    <div class="option-title">Semester</div>
```

Comment:
What about using the "Term" instead of "Semester"? Our school is on the quarter system so we do not have semesters. (summer, autumn, winter, spring) This may be a bigger question, but it would be nice to switch to a more generic term here.